contentcuration/contentcuration/settings.py

```diff
@@ -121,6 +121,7 @@
 MIDDLEWARE = (
     # 'django.middleware.cache.UpdateCacheMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
+    #'contentcuration.middleware.StudioLocaleMiddleware.StudioLocaleMiddleware',
```

Comment:
Should just remove this rather than commenting it out, especially as the file doesn't exist!